src/test/java/org/broadinstitute/hellbender/utils/UtilsUnitTest.java

```diff
@@ -731,4 +731,153 @@ public void testGetDuplicatedItems(final Collection<?> collection, final Set<?>
+                    new Object[] { stringData, "T",   Arrays.asList( stringData.split("T"  ) ) },
+                    new Object[] { stringData, "9",   Arrays.asList( stringData.split("9") ) },
+                    new Object[] { allDelimiterTestString,    ":",   Arrays.asList( allDelimiterTestString.split(":") ) },
+                    new Object[] { frontDelimiterTestString,  ":",   Arrays.asList( frontDelimiterTestString.split(":") ) },
+                    new Object[] { middleDelimiterTestString, ":",   Arrays.asList( middleDelimiterTestString.split(":") ) },
+                    new Object[] { backDelimiterTestString ,  ":",   Arrays.asList( backDelimiterTestString.split(":") ) },
+                    new Object[] { fullDelimiterTestString ,  ":",   Arrays.asList( fullDelimiterTestString.split(":") ) },
+                    new Object[] { allDelimiterTestString.replace(":", "TS"),    "TS",   Arrays.asList( allDelimiterTestString.replace(":", "TS").split("TS") ) },
+                    new Object[] { frontDelimiterTestString.replace(":", "TS"),  "TS",   Arrays.asList( frontDelimiterTestString.replace(":", "TS").split("TS") ) },
+                    new Object[] { middleDelimiterTestString.replace(":", "TS"), "TS",   Arrays.asList( middleDelimiterTestString.replace(":", "TS").split("TS") ) },
+                    new Object[] { backDelimiterTestString.replace(":", "TS") ,  "TS",   Arrays.asList( backDelimiterTestString.replace(":", "TS").split("TS") ) },
+                    new Object[] { fullDelimiterTestString.replace(":", "TS") ,  "TS",   Arrays.asList( fullDelimiterTestString.replace(":", "TS").split("TS") ) }
+                )
+        );
+
+        // Create test cases for words:
+        for ( final String delim : wordsToSplitOn ) {
+            testCases.add( new Object[] { stringData, delim, Arrays.asList( stringData.split(delim) ) } );
+        }
+
+        // Create test cases for repeated substrings:
+        for ( final String delim : repeatedSubstringsToSplitOn ) {
+            testCases.add( new Object[] { stringData, delim, Arrays.asList( stringData.split(delim) ) } );
+        }
+
+        return testCases.iterator();
+    }
+
+    @DataProvider
+    private Iterator<Object[]> provideDataForTestUtilsSplitStringExhaustively() {
+
+        // Length that we want to test through:
+        final int maxStringLength = 7;
+
+        // Create single character delimiter strings:
+        final String singleCharDelimiter = "o";
+        final List<String> singleCharTestStrings = Arrays.asList("X", singleCharDelimiter);
+        final List<List<String>> exhaustiveListsForSingleChar = Utils.makePermutations( singleCharTestStrings, maxStringLength, true );
+
+        // Create multi-character delimiter strings:
+        final String multiCharDelimiter = "oz";
+        // Must include the individual characters of the multi-char delimiter here:
```

Comment:
Can you explain why?

Reply:
The call to `Utils.makePermutations` takes a list of objects and permutes them, so I'm passing individual characters in to create the permuted string.

I've added a comment as well.